compiler/src/iree/compiler/Codegen/Dialect/VectorExt/IR/VectorExtOps.td

```diff
@@ -40,7 +40,8 @@ def IREEVectorExt_ToLayoutOp : IREEVectorExt_PureOp<"to_layout", [
   let arguments = (ins
     AnyShaped:$input,
     VectorLayoutInterface:$layout,
-    DefaultValuedAttr<UnitAttr, "false">:$shared_memory_conversion
+    DefaultValuedAttr<UnitAttr, "false">:$shared_memory_conversion,
+    OptionalAttr<AnyAttr>:$intrinsic
```

Comment:
nit: rename to mma_kind for consistency. Also, this should be MMAAttrInterface.

Reply:
I tried using MMAATTRinterface before butbhar some cmake (gpu to vectorext) cyclic dependencies. I'll give it one more try and see.

Reply:
Ah... that's weird. AnyAttr is fine for now then, we can change it once we remove dependencies from gpu to vectorext. A TODO about cyclic dependencies would be good here.

Reply:
sg!

Reply:
done!